Sources/PeripheryKit/Results/Baseline.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/// A baseline set of declarations that are excluded from results.
+public enum Baseline: Codable {
+    case v1(usrs: Set<String>)
+
+    var usrs: Set<String> {
```

Comment:
Because this is a set, two otherwise identical baselines will not be identical - customizing codability for an enum is annoyingly awkward, but its really nice to just be able to checksum two baselines to see if they've changed.

Reply:
I've updated this to sort a sorted set instead. Comparing with a baseline should be trivial now, at least for the same version.